templates/results.hdbs

```diff
@@ -2,46 +2,76 @@
   {{#if this}}
```

Comment:
I don't think it's indispensable now but it would be good whenever you decide to do it. I have never worked on this particular app so I am selfishly less worried but it may happen some day!